snuba/datasets/cdc/groupassignee.py

```diff
@@ -1,6 +1,5 @@
 from snuba.datasets.dataset import Dataset
-from snuba.datasets.entities import EntityKey
-from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.entities.factory import EntityKey, get_entity
```

Comment:
Why do we need the entity factory instead of just importing directly here?

Reply:
Because Datasets are served using a factory, I assumed we'd want to do the same with Entities.

Reply:
I can kind of see the reason for that since datasets need to be looked up by name for querying and for some CLI commands. But I don't know if that would be the case for entity exactly since the entities that can be selected depend on the dataset that is picked.

Reply:
At some point entities will also be looked up by name in the query, although you're right it will be constrained by the Dataset. If there is no good reason to have the factory then I'll remove it. Less code is always better.